db/schema.rb

```diff
@@ -193,12 +193,12 @@
     t.uuid "role_id"
     t.string "language", null: false
     t.string "reset_digest"
-    t.datetime "reset_sent_at", precision: nil
+    t.datetime "reset_sent_at"
     t.boolean "verified", default: false
     t.string "verification_digest"
-    t.datetime "verification_sent_at", precision: nil
+    t.datetime "verification_sent_at"
     t.string "session_token"
-    t.datetime "session_expiry", precision: nil
+    t.datetime "session_expiry"
     t.integer "status", default: 0
     t.index ["email", "provider"], name: "index_users_on_email_and_provider", unique: true
     t.index ["reset_digest"], name: "index_users_on_reset_digest", unique: true
```

Comment:
This shouldn't be included

Reply:
the change should not be included, or the file itself?